.circleci/config.yml

```diff
@@ -156,3 +156,40 @@ workflows:
     jobs:
       - test-android
       - deploy-android: *release-tags
+  
+  android-deploy-on-tag:
+    jobs:
+      - test-android:
+          filters:
+            tags:
+              only: /^(\d+\.)(\d+\.)(\d+)$/
+            branches:
+              ignore: /.*/
+      - deploy-android:
+          requires:
+            - test-android
+          filters:
+            tags:
+              only: /^(\d+\.)(\d+\.)(\d+)$/
+            branches:
+              ignore: /.*/
+  
+  android-deploy-amazon:
+    jobs:
+      - test-android:
+          filters:
+            branches:
+              only: amazon
+      - hold:
+          type: approval
+          requires:
+            - test-android
```

Comment:
why does this one require approval vs the others?

Reply:
Same as the other comment. We were triggering releases using the approval step, I don't remember why. I removed it because it's not needed